### ISSUE TITLE
False positive aria-allowed-attr for aria-expanded on menuitemcheckbox and menuitemradio

### ISSUE THREAD

**Issue by smhigley** [dequelabs/axe-core#3992]
### Product

axe-core

### Latest Version

- [X] I have tested the issue with the latest version of the product

#### Expectation
`aria-expanded` is an allowed attribute on both menuitemcheckbox and menuitemradio (ref: https://w3c.github.io/aria/#menuitemcheckbox).

#### Actual
axe-core currently returns an aria-allowed-attr failure when it is added to either role.

#### How to Reproduce
Simplest repro: https://jsfiddle.net/fu9ve7gp/show

#### Additional context
Looks like the issue is here: https://github.com/dequelabs/axe-core/blob/develop/lib/standards/aria-roles.js#L377-L396

#3343 would fix it along with some other `aria-expanded` support mismatches, but since it's over a year old I wasn't sure if it was still active.

**Comment by straker** [comment]
Thanks for the issue. That particular pr was put on hold until we could figure out how to inform users of deprecated allowances. However, if you'd like to create a pr that just adds `aria-expanded` to the needed roles, that would be great.

**Comment by smhigley** [comment]
@straker sure, I'd be happy to!
